macros/source_for_test.sql

```diff
@@ -0,0 +1,7 @@
+{% macro source_for_test(schema, table) %}
+
+    {% if target.name != "prod" %} {{ source(target.schema, table) }}
+    {% else %} {{ source(schema, table) }}
+    {% endif %}
+
+{% endmacro %}
```

Comment:
why is this needed?

In case this is caused by the discrepancy on the settings for databases, I think we can remove it

Reply:
Correct. The models use sources like `from {{ source("xapi", "xapi_events_all") }}` but dbt creates the seed tables in the database specified in the profiles.yml file which is different depending on if you are running locally or in CI